### ISSUE TITLE
Result on UCF101

### ISSUE THREAD

**Issue by SwithunXiaodong** [RI-CH/SlowFastNetworks#1]
Thank you for sharing your implementation, have you evaluated the net in the UCF101? how about the result?

**Comment by RI-CH** [comment]
@SwithunXiaodong  UCF101 is a small dataset, so model trained on this dataset from scratch is easily overfitting. 
The accuracy on the validation dataset much lower than on the training dataset. After 28 epoches, validation accuracy is 42%, training accuracy is 97%.

"ConvNet Architecture Search for Spatiotemporal Feature Learning" discussed this issue.

**Comment by RI-CH** [comment]
@SwithunXiaodong  UCF101 is a small dataset, so model trained on this dataset from scratch is easily overfitting. 
The accuracy on the validation dataset is much lower than on the training dataset. After 28 epochs, validation accuracy is 42%, while training accuracy is 97%.

"ConvNet Architecture Search for Spatiotemporal Feature Learning" discussed this issue.
